cli/commands.go

```diff
@@ -7,7 +7,7 @@ var (
 		{
 			Name:      "setup",
 			ShortName: "s",
-			Usage:     "",
+			Usage:     "Setup initialize the specified collection, it's required befor using a collection.",
```

Comment:
~~Setup~~ Initialize the specified collection, it's required befor`e` using a collection.